### ISSUE TITLE
Статья на хабре

### ISSUE THREAD

**Issue by Hedgehogues** [headfire/desk#1]
Привет. Это Ваша статья на хабре ?https://habr.com/ru/post/518378/

**Comment by headfire** [comment]
Привет! Да это я написал.

**Comment by Hedgehogues** [comment]
Подскажите, Вы работали с pythonOCC? Есть ли у Вас наработки? Очень мало информации на этот счёт.

**Comment by headfire** [comment]
Да, я работал с pythonOCC, Собственно об этом и статья. Наработки в виде  [репозитария](https://github.com/headfire/crpoint). Более-менее я в этой теме разобрался. А что Вас интересует конкретно?

**Comment by Hedgehogues** [comment]
У меня есть задача, которая связана с обработкой большого количества деталей. Например датасет ABC. Но, увы, даже понять, из каких примитивов состоит деталь (цилиндры, кубики, сферы и т.п.) оказалась не самой тривиальной. В связи с этим, я ищу людей, которые разбиратся. Их не так много особенно в рунете

**Comment by headfire** [comment]
Я могу попробовать поработать в этой теме. На каких условиях будет происходить рабочий процесс?
